.github/workflows/promote_to_branch.yml

```diff
@@ -0,0 +1,78 @@
+name: Run Test on push to branch
+
+on:
+  push:
+    branches:
+      - '*'
+      - '!master'
+
+jobs:
+
+  # This action (dorny/paths-filter) allows us to selectively run jobs based on
+  # the files that have been updated, as in on.push.paths above.
+  # Saves us from doing unnecessary builds.
+  # ---------------------------------------------------------------------------
+  
+  api_test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./api
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "12"
+      - name: Install dependencies
+        run: npm ci
+      - name: Unit tests w/ coverage
+        run: npm run test:coverage
+      - name: LINTing
+        run: npm run test:lint
+      - name: OpenAPI Schema
+        run: npm run test:schema
+      - name: Upload coverage report
```

Comment:
We haven't been using this coverage report system, is there a reason you have chosen to include it?